packages/components/psammead-topic-tags/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "@bbc/psammead-topic-tags",
+  "version": "1.0.0",
+  "main": "dist/index.js",
+  "module": "esm/index.js",
+  "sideEffects": false,
+  "description": "Contains topic links",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/BBC-News/psammead/tree/latest/packages/components/psammead-topic-tags"
+  },
+  "author": {
+    "name": "Psammead Maintainers",
+    "email": "PsammeadMaintainers@bbc.co.uk"
+  },
+  "license": "Apache-2.0",
+  "bugs": {
+    "url": "https://github.com/bbc/psammead/issues"
+  },
+  "homepage": "https://github.com/BBC-News/psammead/blob/latest/packages/components/psammead-topic-tags/README.md",
+  "dependencies": {
+    "@bbc/psammead-styles": "^7.2.1"
+  },
+  "peerDependencies": {
+    "react": "&gt;=16.9.0"
```

Comment:
Is that "`"&gt;=16.9.0"`" GitHub being weird? 🤔

Reply:
Huh, very odd, this is on my local branch too. It must have come from either `npm run create:package` or VSCode playing up somehow, good spot.